.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,55 @@
+name: "Benchmark workflow"
+on:
+  # Push and update benchmarks when a commit is merged into master.
+  push:
+    branches:
+      - master
+  # Trigger benchmark test on this PR's commit against master.
+  pull_request:
+    branches:
+      - master
+permissions:
+  # Push benchmark performance graph to gh-pages branch.
+  contents: write
+  deployments: write
+jobs:
+  benchmark:
+    name: Performance regression check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version-file: 'go.mod'
+      - run: go version
+      - name: Build go-algorand
+        run: scripts/travis/build.sh
+      - name: Run benchmark
+        run: go test ./data/transactions/logic -bench 'BenchmarkUintMath' | tee benchmark_output.txt
+      - name: Download previous benchmark data
+        uses: actions/cache@v1
+        with:
+          path: ./cache
+          key: ${{ runner.os }}-benchmark
+      - name: Push benchmark result
+        if: github.event_name == 'push'
+        uses: benchmark-action/github-action-benchmark@v1
+        with:
+          name: Go Benchmark
+          tool: 'go'
+          output-file-path: benchmark_output.txt
+          # Where the previous data file is stored
```

Comment:
@algochoi Minor - I think the highlighted comment is inaccurate.  If you'd like to retain a comment here (rather than removal), I'm open to it.

I suspect the comment is a copy-paste holdover.

```suggestion

```